include/pstore/core/file_header.hpp

```diff
@@ -223,6 +223,7 @@ namespace pstore {
         /// This crc is used to ensure that the fields from #signature1 to #sync_name are not
         /// modified.
         std::uint32_t crc = 0;
+        std::uint32_t unused1 = 0;
 
         std::uint64_t unused2 = 0;
 
```

Comment:
After adding unused1, should we remove unused2?

Reply:
Thanks Maggie. I had presumed that because the fields were nailed to hard coded values the unused2 space was needed for some reason. 

I will remove it, change the hard coded constants and recommit.